draco/analysis/qstack.py

```diff
@@ -0,0 +1,148 @@
+# === Start Python 2/3 compatibility
+from __future__ import absolute_import, division, print_function, unicode_literals
+from future.builtins import *  # noqa  pylint: disable=W0401, W0614
+from future.builtins.disabled import *  # noqa  pylint: disable=W0401, W0614
+
+# === End Python 2/3 compatibility
+
+import h5py
+import numpy as np
+import healpy as hp
+from mpi4py import MPI
+
+from caput import mpiarray, config, mpiutil, pipeline
+from cora.util import units
+from ch_util import tools, ephemeris
+from drift.telescope import cylbeam
+
+from ..util.tools import invert_no_zero
+from ..util._fast_tools import beamform
+from ..core import task, containers, io
+
+# Constants
+NU21 = units.nu21
+C = units.c
+
+
+class QuasarStack(task.SingleTask):
+    """
```

Comment:
Add a docstring. Also, maybe call it `SourceStack` as we'll surely work with other types of sources.